client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1127,6 +1127,15 @@ Undocumented.prototype.readA8C = function( query, fn ) {
 	return this.wpcom.req.get( '/read/a8c', query, fn );
 };
 
+Undocumented.prototype.readConversations = function( query, fn ) {
```

Comment:
funny, I strongly dislike this file. :) Too much cross-pollination and unnecessary junk. Though maybe with tree-shaking it'll be ok.